src/modules/remote.json

```diff
@@ -0,0 +1,151 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+      "title": "Remote",
+      "description": "A module for managing Remote keys, paring and programming.",
+      "version": "0.0.0"
+    },
+    "methods": [
+      {
+        "name": "state",
+        "summary": "Read-only property method for the remote control state. This is a getter for the state or a way to subscribe to changes to the state.",
+        "params": [],
+        "result": {
+          "name": "remoteList",
+          "summary": "Contains a list of remotes controls paired to the device.",
+          "schema": {
+            "$ref": "#/components/schemas/RemoteControlState"
+          }
+        },
+        "examples": [
+          {
+            "name": "Remote State Example",
+            "params": [],
+            "result": {
+              "remoteList": [
+                {
+                  "model": "PR2",
+                  "batteryPercentage": "70"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "name": "startPairing",
+        "summary": "Setup the device to allow Remote control to be paired on the next keypress.",
+        "params": [],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
+          }
+        },
+        "examples": [
+          {
+            "name": "Remote State Examples",
+            "params": [],
+            "result": {
+              "pairingState": "pairingRemote"
+            }
+          }
+        ]
+      },
+      {
+        "name": "onRemotePairing",
+        "tags": [
+          {
+            "name": "event"
+          }
+        ],
+        "summary": "Listen for events when the remote control is pairing.",
+        "params": [
+          {
+            "name": "listen",
+            "required": true,
+            "summary": "whether to listen, or stop listening",
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "result": {
+          "name": "remotePairState",
+          "summary": "The state of currently paired remote",
+          "schema": {
+            "$ref": "#/components/schemas/RemotePairState"
+          }
+        },
+        "examples": [
+          {
+              "name": "Default Example",
+              "params": [
+                  {
+                      "name": "listen",
+                      "value": true
+                  }
+              ],
+              "result": {
+                  "name": "Default Result",
+                  "value": {
+                      "event": "Remote.onRemotePairing",
```

Comment:
The SDK keeps track of the event via RPC ids. None of our existing events include the name of the event in the response.

It's not a bad idea, but we'd have to bring it up holistically, not as a one-off on this new event.

I suggest you remove it for now, and write up a proposal for all events if you think it's warranted.